activiti-core/activiti-engine/src/main/java/org/activiti/engine/impl/db/DbSqlSession.java

```diff
@@ -1169,6 +1169,10 @@ public boolean isTablePresent(String tableName) {
                 catalog = null;
             }
 
+            if (catalog == null && ProcessEngineConfigurationImpl.DATABASE_TYPE_MYSQL.equalsIgnoreCase(databaseType)) {
```

Comment:
Did you try setting the catalog value by using `ProcessEngineConfiguration.setDatabaseCatalog` ? Will this change still work with previous MySQL versions i.e. 5.x?

Reply:
Thanks for you anwser. just config databaseCatalog like this `<property name="databaseCatalog" value="activiti_7" />`,but  the value sholud config the same as jdbcUrl, Why not get catalog from connection?

[https://github.com/shootercheng/SpringBoot2-Activiti7/blob/master/src/test/java/db/mysql/JdbcTest.java](https://github.com/shootercheng/SpringBoot2-Activiti7/blob/master/src/test/java/db/mysql/JdbcTest.java)